spec/install/gems/standalone_spec.rb

```diff
@@ -75,10 +91,19 @@
       G
     end
 
-    it "generates a bundle/bundler/setup.rb with the proper paths", :rubygems => "2.4" do
-      extension_line = File.read(bundled_app("bundle/bundler/setup.rb")).each_line.find {|line| line.include? "/extensions/" }.strip
-      expect(extension_line).to start_with '$:.unshift "#{path}/../#{ruby_engine}/#{ruby_version}/extensions/'
-      expect(extension_line).to end_with '/very_simple_binary-1.0"'
+    it "loads gems correctly", :rubygems => "2.4" do
+      testrb = String.new <<-RUBY
+        $:.unshift File.expand_path("bundle")
+        require "bundler/setup"
+
+        require 'very_simple_binary'
+        puts (Gem.loaded_specs['very_simple_binary'] || Gem::Specification.new).name
```

Comment:
please `puts VerySimpleBinaryInC` and verify the output, which will make sure that the correct file has been loaded